test/test_model.py

```diff
@@ -105,6 +107,23 @@ def test_average_model():
     assert np.all(tmodel_2000.predict([0, 0, 0]) == 1100)
 
 
+def test_gp_model(datadir):
+    dwi = DWI.from_filename(datadir / "dwi.h5")
+
+    kernel = DotProduct() + WhiteKernel()
```

Comment:
It does look like kernel is an object not a type.

Reply:
According to the scikit-learn example,
https://scikit-learn.org/stable/auto_examples/gaussian_process/plot_gpr_noisy_targets.html#example-with-noise-free-target

It is an object, yes, so not sure what you want to change in here